src/styles/BgCircle.css

```diff
@@ -0,0 +1,77 @@
+
+:root {
+    --Color1: lightblue;
+    --Color2: lightgreen;
+    --Color3: red;
+    --Color4: yellow;
+    --circleSize: 85vw;
+    --pos1-left: 90vw;
+    --pos1-top: 0vh;
+    --pos2-left: 0vw;
+    --pos2-top: 0vh;
+    --pos3-left: 0vw;
+    --pos3-top: 100vh;
+    --pos4-left: 90vw;
+    --pos4-top: 100vh;
+  }
+.bg{
```

Comment:
There was a PR by @Arjun-Dagar , solves the responsiveness problem with the logo. Do check it out as well. Rn the logo circle will not intersect the bigger circle on medium sized devices.

Reply:
I have checked that pr, for responsiveness for logo, we have to adopt a different approach as used in BG circle, I am working on that